ql/src/java/org/apache/hadoop/hive/ql/exec/repl/bootstrap/load/table/LoadPartitions.java

```diff
@@ -98,6 +104,17 @@ private String location() throws MetaException, HiveException {
     }
   }
 
+  private void createTableReplLogTask() throws SemanticException {
+    ReplStateLogWork replLogWork = new ReplStateLogWork(replLogger,
+                                            tableDesc.getTableName(), tableDesc.tableType());
+    Task<ReplStateLogWork> replLogTask = TaskFactory.get(replLogWork, context.hiveConf);
+    if (tracker.tasks().isEmpty()) {
+      tracker.addTask(replLogTask);
+    } else {
+      ReplLoadTask.dependency(tracker.tasks(), replLogTask);
```

Comment:
tracker is not taking into account this new task added.

Reply:
I think, it is ok, because this is the last task being added in the current table's DAG. When the caller adds this to loadTaskTracker during post-processing, it will be accounted.